.eslintrc.js

```diff
@@ -1,25 +1,29 @@
 module.exports = {
   root: true,
 
-  extends: [
-    '@metamask/eslint-config',
-    '@metamask/eslint-config/config/nodejs',
-  ],
+  extends: ['@metamask/eslint-config'],
 
-  parserOptions: {
-    ecmaVersion: 2017,
-    ecmaFeatures: {
-      arrowFunctions: true,
-      classes: true,
+  overrides: [
+    {
+      files: ['*.js'],
+      extends: ['@metamask/eslint-config-nodejs'],
+      parserOptions: {
+        sourceType: 'module',
```

Comment:
This seems wrong, this module doesn't use ES6 modules

Reply:
This might also allow removing the second override block.

Reply:
Sigh. I always thought `parserOptions: 'script'` just meant that there weren't any imports or exports of any kind (you know... like a script). It helps if you read the docs I guess.

Fixed in [898f24b](https://github.com/MetaMask/eth-json-rpc-infura/pull/55/commits/898f24b59b5fabb3aeaed7242e961cb7eb54e11a).